PSICover/PSICover.cs

```diff
@@ -163,17 +164,47 @@ class Analyzer {
          var code = File.ReadAllLines (file);
          for (int i = 0; i < code.Length; i++)
             code[i] = code[i].Replace ('<', '\u00ab').Replace ('>', '\u00bb');
+         int hitCnt = 0;

```

Comment:
Minor style guideline. I generally use lowercase c as a prefix to indicate a count. So this variable would be called `cHits`.